tmt/utils.py

```diff
@@ -4856,19 +4864,53 @@ def git_clone(
 
     if can_change:
         url = clonable_git_url(url)
+
+    def get_env(env: str) -> Optional[int]:
```

Comment:
could be easier with a default argument

you can then pass the DEFAULT_GIT_CLONE_* and not have to check for None